src/flask_httpauth.py

```diff
@@ -254,9 +254,13 @@ def authenticate(self, auth, stored_password):
 
 
 class HTTPDigestAuth(HTTPAuth):
-    def __init__(self, scheme=None, realm=None, use_ha1_pw=False):
+    def __init__(self, scheme=None, realm=None, use_ha1_pw=False, qop=None):
         super(HTTPDigestAuth, self).__init__(scheme or 'Digest', realm)
         self.use_ha1_pw = use_ha1_pw
+        if qop and qop == 'auth': #only support qop=auth
+            self.qop = qop
+        else:
```

Comment:
I would prefer that when `qop` is set to an unsupported value there is a `ValueError` exception clearly indicating that this is unsupported.